### PR TITLE
[zh-cn] Localization blog: 2024-04-23-recursive-read-only-mounts.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/blog/_posts/2024-04-23-recursive-read-only-mounts.md

### CODE REVIEW

**Comment by my-git9** [comment]
The preview：https://deploy-preview-46127--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2024/04/23/recursive-read-only-mounts/

File: content/zh-cn/blog/_posts/2024-04-23-recursive-read-only-mounts.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+<!--
+Read-only volume mounts have been a feature of Kubernetes since the beginning.
+Surprisingly, read-only mounts are not completely read-only under certain conditions on Linux.
+As of the v1.30 release, they can be made completely read-only,
+with alpha support for _recursive read-only mounts_.
+-->
+只读卷挂载从一开始就是 Kubernetes 的一个特性。
+令人惊讶的是，在 Linux 上的某些条件下，只读挂载并不是完全只读的。
+从 v1.30 版本开始，它们可以完全只读，并具有对**递归只读安装**的 Alpha 支持。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-04-23-recursive-read-only-mounts.md:29]:
```suggestion
从 v1.30 版本开始，这类卷挂载可以被处理为完全只读；v1.30 为**递归只读挂载**提供 Alpha 支持。
```

File: content/zh-cn/blog/_posts/2024-04-23-recursive-read-only-mounts.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+<!--
+## How to get involved?
+
+This feature is driven by the SIG Node community. Please join us to connect with
+the community and share your ideas and feedback around the above feature and
+beyond. We look forward to hearing from you!
+-->
+## 如何参与？
+
+此特性由 SIG node 社区推动。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-04-23-recursive-read-only-mounts.md:195]:
```suggestion
此特性由 SIG Node 社区推动。
```

**Pull request merged**